src/ehttpc.erl

```diff
@@ -209,6 +210,7 @@ handle_info({gun_down, Client, _, Reason, KilledStreams, _}, State = #state{clie
     {noreply, State#state{gun_state = down, requests = NRequests}};
 
 handle_info({'DOWN', MRef, process, Client, Reason}, State = #state{mref = MRef, client = Client, requests = Requests}) ->
+    ?LOG(warning, "The process of gun exited due to ~p", [Reason]),
```

Comment:
```suggestion
    ?LOG(warning, "gun_process_exited due to ~p", [Reason]),
```

Reply:
I think it is more reasonable to appear as a normal statement in the log, and adding an underscore will make it look like the information returned from a certain function.